azure/functions_worker/bindings/__init__.py

```diff
@@ -1,25 +1,14 @@
 from .context import Context
 from .meta import check_input_type_annotation
 from .meta import check_output_type_annotation
-from .meta import is_binding, is_trigger_binding
+from .meta import is_trigger_binding
 from .meta import from_incoming_proto, to_outgoing_proto
 from .out import Out
 
-# Import type implementations and converters
-# to get them registered and available:
-from . import blob  # NoQA
-from . import cosmosdb  # NoQA
-from . import eventgrid  # NoQA
-from . import eventhub  # NoQA
-from . import http  # NoQA
-from . import queue  # NoQA
-from . import servicebus  # NoQA
-from . import timer  # NoQA
-
 
 __all__ = (
     'Out', 'Context',
-    'is_binding', 'is_trigger_binding',
+    'is_trigger_binding',
```

Comment:
why do you need 'is_trigger_binding'?

Reply:
It is used to distinguish regular input bindings from trigger bindings, because there are differences in handling these (metadata, etc.)